### PR TITLE
Feature - IJ-123 User Journals Dashboard Updates

### PR DESCRIPTION
**Pull request by garethdandrews**
Removed the Journal Entries dashboard, and redirect users to 'add new' page if no journal entries are present. Added subheading stats to journal entries index.

I've got two buttons to add a new journal entries. Let me know which one I should keep/remove.

journal_entries_index

### CODE REVIEW

**Comment by IeuanSkinner** [comment]
Not a fan of having 2 buttons to do the same function within the same view (i.e. the big square new entry button and the smaller green new entry button). My preference is to keep the green button and lose the big square one.

**Comment by garethdandrews** [comment]
> Not a fan of having 2 buttons to do the same function within the same view (i.e. the big square new entry button and the smaller green new entry button). My preference is to keep the green button and lose the big square one.

@IeuanSkinner The plan is to remove one of them. I couldn't decide which I preferred.

**Comment by phil-6** [comment]
On Buttons:
I don't think either is quite right at the moment. 
I think my preference would be smaller "primary" button, directly under the title.

File: app/assets/stylesheets/buttons.scss [hunk 1]
```diff
@@ -87,5 +87,9 @@ body#overboot {
     &.two {
       max-width: 400px;
     }
+
+    &.one {
+      max-width: 160px;
+    }
```
Comment (phil-6) [inline app/assets/stylesheets/buttons.scss:93]: Can this go above "two" please - if its still needed.

**Review by phil-6 — COMMENTED** [review]
LGTM, just the button changes. :+1:

**Comment by garethdandrews** [comment]
Style updates as per comments

**Review by AlexW555 — APPROVED** [review]
LGTM: In regards to the button, there isn't a consistent style for where an "add new" button should go or look like. But I like the green button look here

**Comment by IeuanSkinner** [comment]
@garethdandrews when you update the UI in later commits can you be sure to include screenshots 👍

**Pull request merged**